src/main/java/seedu/address/model/person/Room.java

```diff
@@ -12,6 +12,7 @@
     public static final String MESSAGE_CONSTRAINTS =
             "Room numbers should only contain numbers, and it should be 3 digits long";
     public static final String VALIDATION_REGEX = "\\d{3}";
+    private static int roomPref = 420;
```

Comment:
Would it be better to declare it as a final variable? As it behaves as a constant.